storage/src/adb/immutable/sync/mod.rs

```diff
@@ -183,15 +127,11 @@ where
                 .await
                 .map_err(crate::adb::Error::from)?;
 
-            // Compute next append location based on logical locations within [lower_bound, upper_bound]
-            let size = compute_size(
-                &variable_journal,
-                config.log_items_per_section,
-                lower_bound,
-                upper_bound,
-            )
-            .await
-            .map_err(crate::adb::Error::from)?;
+            // Get the size of the journal
+            let size = get_size(&variable_journal, config.log_items_per_section.get()).await?;
```

Comment:
Will be great to keep size cached: https://github.com/commonwarexyz/monorepo/issues/1590